modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -271,6 +271,67 @@ bond=bond0:em1,em2:mode=active-backup
 vlan=bond0.100:bond0
 ----
 
+[id="bonding_multiple_sriov_network_interfaces_to_dual_port_{context}"]
+[discrete]
+=== Bonding multiple SR-IOV network interfaces to a dual port NIC interface
+
+[IMPORTANT]
+====
+Support for Day 1 operations associated with enabling NIC partitioning for SR-IOV devices is a Technology Preview feature. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+Optional: You can bond multiple SR-IOV network interfaces to a dual port NIC interface by using the `bond=` option.
+
+.Prerequisties
+
+* On each node, you need to:
+
+. Create the SR-IOV virtual functions (VFs) following the guidance in link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_virtualization/managing-virtual-devices_configuring-and-managing-virtualization#managing-sr-iov-devices_managing-virtual-devices[Managing SR-IOV devices]. Follow the procedure in section 13.7.2.
+
+. Create the bond, attach the desired VFs to the bond and set the bond link state up following the guidance in link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_networking/configuring-network-bonding_configuring-and-managing-networking[Configuring network bonding
+]. Follow any of the described procedures to create the bond.
+
+The following examples illustrate the syntax you need to use:
+
+* The syntax for configuring a bonded interface is: `bond=name[:network_interfaces][:options]`
++
+_name_ is the bonding device name (`bond0`), _network_interfaces_
+represents the virtual functions (VFs) by their known name in the kernel and shown in the output of the `ip link` command.(eno1f0,eno2f0), and _options_ is a comma-separated list of bonding options. Enter `modinfo bonding` to see available options.
+
+* When you create a bonded interface using `bond=`, you must specify how the IP address is assigned and other information for the bonded interface.
+
+* To configure the bonded interface to use DHCP, set the bond's IP address to `dhcp`. For example:
++
+[source,terminal]
+----
+bond=bond0:eno1f0,eno2f0:mode=active-backup
+ip=bond0:dhcp
+----
+
+* To configure the bonded interface to use a static IP address,
+enter the specific IP address you want and related information. For example:
+ifndef::ibm-z[]
++
+[source,terminal]
+----
+bond=bond0:eno1f0,eno2f0:mode=active-backup
+ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:bond0:none
+----
+endif::ibm-z[]
+ifdef::ibm-z[]
+
+[source,terminal]
```

Comment:
This source block and following sentence aren't indented properly on the IBM Z docs:

![image](https://user-images.githubusercontent.com/7443887/220724819-2990f711-c7d9-4c20-b5bc-5cb9bc81f330.png)

Reply:
@bergerhoffer where can I preview the IBM Z docs ?

Reply:
@kquinn1204 I don't remember off the top of my head, but I clicked around the different assemblies in your preview to find an IBM Z one and found this.